preinstall.sh

```diff
@@ -73,12 +242,16 @@ case $(uname -s) in
     echo "Detected Mac OS"
     if type "brew" > /dev/null; then
       echo "Installing libvips via homebrew"
-      brew install homebrew/science/vips --with-webp --with-graphicsmagick
+      if [ $enable_openslide -eq 1 ]; then
+        brew install homebrew/science/vips --with-webp --with-graphicsmagick --with-openslide
+      else
+        brew install homebrew/science/vips --with-webp --with-graphicsmagick
+      fi
     elif type "port" > /dev/null; then
       echo "Installing libvips via MacPorts"
       port install vips
```

Comment:
Can you check what happens if openslide is enabled? Does vips gets installed with openslide support or not? I don't have Macports so I cannot test.